### PR TITLE
fix: typo in reusable rubric copy

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** 

fix a small typo from `effect` to `affect`

JIRA: [AU-363](https://openedx.atlassian.net/browse/AU-363)

**What changed?**

image

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by leangseu-edx** [comment]
I thought it was too small of a change to bump the version.

**Comment by jansenk** [comment]
>  If you just do a release with my commit's release number, it will catch both changes.

Funnily enough that was also a typo fix, so 3.7.3 can just be 3.7.3: Fix Typos
